bin/convert_ismapper_to_vcf.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
+
+import argparse
+import csv
+from Bio import SeqIO
+
+
+
+# Define the VCF header
+vcf_header = """##fileformat=VCFv4.2
+##source=ISMapper
+##INFO=<ID=SVTYPE,Number=1,Type=String,Description="Type of structural variant">
+#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO
+"""
+
+# Function to read the reference genome
+def read_reference_genome(reference_file):
+    reference_sequences = {}
+    reference_sequence_length = 0
+    with open(reference_file, 'r') as ref_file:
+        # Read the reference genome using Biopython
+        for record in SeqIO.parse(ref_file, "fasta"):
+            reference_sequences[record.id] = record.seq
+            reference_sequence_length = len(record.seq)
+    return reference_sequences, reference_sequence_length
+
+# Function to read the transposable element information
+def read_transposable_elements(te_file):
```

Comment:
Here's I changed the script derive the `te_info` object from the provided multi-fasta sequence file (see `resources/regions/IS_Queries_Mtb.multi.fasta` file in this branch)

Reply:
would this not require a change to the script so that it pulls the name from the pasta sequence name?

Reply:
> would this not require a change to the script so that it pulls the name from the pasta sequence name?
My friend, doing that to a `pasta` 🍝 would be tough 😉 
 

But jokes apart, yeah the `read_transposable_elements` has been refactored to derive the length of the sequence from the input multi_fasta file so it should work

Reply:
Ok great! Thanks Abhi!